cmd/beacon-chain/storage/options.go

```diff
@@ -0,0 +1,30 @@
+package storage
+
+import (
+	"github.com/prysmaticlabs/prysm/v4/beacon-chain/node"
+	"github.com/prysmaticlabs/prysm/v4/cmd"
+	"github.com/urfave/cli/v2"
+)
+
+var (
+	// StatePath defines a flag to start the beacon chain from a give genesis state file.
+	BlobStoragePath = &cli.PathFlag{
+		Name:  "blob-path",
+		Usage: "Location for blob storage. Default location will be a 'blobs' directory next to the beacon db.",
+	}
```

Comment:
I'll look into that but it could be complicated given that it is based on another flag value - so there would need to be some variant of `Usage` that was lazy evaluated, so that it could access the set value of another flag.

Reply:
OK -- my personal opinion is that it would be better to have the full path in the help text but it doesn't seem very feasible. 

I think your implementation matches how others do it (ex: geth's keystore directory is the same) 

https://github.com/ethereum/go-ethereum/blob/5bf8769fb081c5f17f23b1b55e508789f0a8e632/node/config.go#L74